.github/workflows/ci.yml

```diff
@@ -86,6 +86,7 @@ jobs:
         if: ${{matrix.os == 'ubuntu-latest'}}
       - run: yarn --frozen-lockfile
       - run: yarn build-native-release
+      - run: yarn build
```

Comment:
Yeah, you have to do a build because ESM imports don't resolve to the "source" field or go through babel register. So when we load an ESM plugin, and that loads `@parcel/plugin` or other packages, it'll need to find the compiled version.